3scale/porta
# THREESCALE-8979: Fix the issue with not being able to remove default plan

**What this PR does / why we need it**:

Fix the issue with not being able to remove default plan.

**Which issue(s) this PR fixes** 

Fixes https://issues.redhat.com/browse/THREESCALE-8979

**Verification steps** 

1. Create a product, and an application plan for it
2. Open the application plans list and see that _No plan selected_ is shown for the Default plan
3. in the drop-down select an application plan and click Change plan. The plan is changed and the "success" flash message is shown.
4. Select _No plan selected_ again, and click Change plan.
5. The default plan is unset, the page is reloaded and the "success" flash message is shown.

**Special notes for your reviewer**:

In Flow there was an _ignore_ message explaining why there was an empty string: https://github.com/3scale/porta/pull/2974/files#diff-5360738f38dabacb7a4d30fca57a8de39dd864cae351edae17bfe5c0b70fdae1R27

The ID of the default plan was changed from `''` to `-1` in the Typescript PR: https://github.com/3scale/porta/pull/3054/files
